okhttp/src/jvmMain/kotlin/okhttp3/MediaType.kt

```diff
@@ -107,31 +106,30 @@ class MediaType private constructor(
     @JvmStatic
     @JvmName("get")
```

Comment:
Why not move at the same time to common? There seems to be some additional imports required when moving.

```
import kotlin.jvm.JvmName
import kotlin.jvm.JvmOverloads
```

Reply:
I think I appreciate now, how much easier it is to incrementally migrate file by file! Just removing the java.* imports.

Reply:
Yeah, it's gonna be a bit of a slog.